config/environments/demo.rb

```diff
@@ -13,7 +13,7 @@
   }
   config.action_mailer.default_url_options = { host: 'demo.getyourrefund.org' }
   config.action_mailer.asset_host = config.gyr_url
-  config.offseason = false
+  config.offseason = true
```

Comment:
I'd like to see us set `config.offseason = true` for the test environment, too. That way, the test environment is close to demo/dev.

Maybe that'll be too much of a hassle? OK, maybe forget it.

Reply:
I think that would break some tests, because the whole point of this is to keep testing the "on season flow" and then add tests that check for the offseason.